lib/formatAtRuleParams.js

```diff
@@ -1,13 +1,25 @@
 function formatAtRuleParams (atrule) {
   var params = atrule.params
   var atName = atrule.name
+  var atTypes = [
+    'media',
+    'include',
+    'mixin',
+    'function',
+    'for',
+    'each',
+    'while',
+    'else'
+  ]
 
-   if (atName === 'media' || atName === 'include' || atName === 'mixin') {
+   if (atTypes.indexOf(atName) > -1) {
```

Comment:
I also have a fix coming for Sass' `@if/@else`, but it depends on this code to get merged first.